extra/networkNameMapping.json

```diff
@@ -13,5 +13,6 @@
 	"fyber" : "FairBidSDK",
 	"mytarget" :"myTargetSDK",
 	"snap" :"snap",
-	"mintegral" :"mintegral"
-}
+	"mintegral" :"mintegral",
+	"pangle" : "Ads-Global"
+}
```

Comment:
please add a new line at the end of the file. No new line can cause some parsers to break.